zos_basics/factgathering/ansible.cfg

```diff
@@ -0,0 +1,23 @@
+################################################################################
+# Copyright (c) IBM Corporation 2020, 2021
+################################################################################
+
+################################################################################
+# Minimum required configuration for this playbook.
+# For additional `ansible.cfg` configuration options, review:
+#   https://docs.ansible.com/ansible/latest/reference_appendices/config.html
+#   https://github.com/ansible/ansible/blob/devel/examples/ansible.cfg
+################################################################################
+
+[defaults]
+forks = 25
+display_skipped_hosts = no
```

Comment:
Is display_skipped_hosts and callback_whitelist required for this playbook? Maybe a brief description in the cfg if so.

Reply:
Same for callback_profile_tasks

Reply:
cleaned up the ansible.cfg, those were likely remnants of some experiments